src/item/views/Item.tsx

```diff
@@ -358,14 +363,14 @@ const Item: FunctionComponent<ItemProps> = ({
 												/>
 											</td>
 										</tr>
-										<tr>
-											<th scope="row">Klascement</th>
-											<td>
-												<a href={'http://www.klascement.be/link_item'}>
-													www.klascement.be/link_item
-												</a>
-											</td>
-										</tr>
+										{/*<tr>*/}
```

Comment:
Is dit iets dat achteraf nog geïmplementeerd moet worden of mag dit weg?

Reply:
Dit moet nog geimplementeerd worden, maar VIAA moet nog bepalen van waar die info moet komen. Voorlopig op backlog geplaatst: https://district01.atlassian.net/browse/AVO2-463